swag/client/yfu.py

```diff
@@ -0,0 +1,54 @@
+from disnake.ext import commands
+import disnake
+from swag.id import UserId
+from swag.blocks.yfu_blocks import YfuGenerationBlock
+from utils import GUILD_ID
+
+from .ui.IHS_toolkit import sort_yfus_id
+from .ui.yfu_view import YfuEmbed, YfuNavigation, YfuExchange
+
+
+class YfuCommand(commands.Cog):
+    def __init__(self, swag_client):
+        self.swag_client = swag_client
+
+    @commands.slash_command(name="yfu", guild_ids=[GUILD_ID])
+    async def yfu(self, interaction: disnake.ApplicationCommandInteraction):
+        """Ouvre le menu des ¥fus"""
+        await interaction.response.defer()
+
+    @yfu.sub_command(name="menu")
+    async def open_menu(self, interaction: disnake.ApplicationCommandInteraction):
+        """Ouvre le menu des ¥fus"""
+        # TODO gérer le cas où il n'y a pas de Yfu
+
+        first_yfu_id = sort_yfus_id(
```

Comment:
Do you really need to sort yfus just to select the first one?

Reply:
Yes, because yfus in account are stocked in set. And you must sort a set to get the first one, because as you know, set are unsorted. 

Well, sorting EVERY yfus to get the first one is like overpowered I guess